src/README.md

```diff
@@ -241,14 +241,12 @@ To activate continuous batching please provide additional property to LLMPipelin
 #include "openvino/genai/llm_pipeline.hpp"
 
 int main(int argc, char* argv[]) {
-    ov::AnyMap config;
     ov::genai::SchedulerConfig scheduler_config;
     {
     //fill scheduler_config with custom data if required
     }
-    config[ov::genai::scheduler_config.name()] = scheduler_config;
 
-    ov::genai::LLMPipeline pipe(model_path, "CPU", config);
+    ov::genai::LLMPipeline pipe(model_path, "CPU", ov::genai::scheduler_config(scheduler_config));
```

Comment:
@Wovchena, properties are allowed to be accepted as free standing arguments to LLMPipeline ctor, so this simplification is possible. Please approve explicitly.